clients/client-go/integrationtest/doc.go

```diff
@@ -1,2 +0,0 @@
-// Package integrationtest stores all the integration tests that run against the taskcluster cluster client
-package integrationtest
```

Comment:
Did you mean to delete this file? I see the text has a mistake, but its purpose is to provide an overview of the package for godoc. See [The Go Blog - Godoc: documenting Go code](https://blog.golang.org/godoc-documenting-go-code):

> ... That package uses another convention for packages that need large amounts of introductory documentation: the package comment is placed in its own file, `doc.go`, which contains only those comments and a package clause.